apps/web/src/app/(private)/datasets/generate/GenerateDatasetContent.tsx

```diff
@@ -69,25 +65,18 @@ export function GenerateDatasetContent({
   }
 
   const {
-    runAction: runPreviewAction,
-    done: previewDone,
-    isLoading: previewIsLoading,
+    execute: runPreviewAction,
+    isPending: previewIsLoading,
     error: previewError,
-  } = useStreamableAction<typeof generateDatasetPreviewAction>(
-    generateDatasetPreviewAction,
-    async (event, data) => {
-      if (
-        event === StreamEventTypes.Latitude &&
-        data.type === LegacyChainEventTypes.Complete
```

Comment:
The problem is that the data returned by the SDK is different now, and this code was not changed.